### PR TITLE
Added SideKick browser support for the browser bookmark plugin and bu…

### PR DESCRIPTION
**Pull request by jatisto**
Added SideKick browser support for the browser bookmark plugin and bugfix startup error

### CODE REVIEW

**Comment by oliverschwendener** [comment]
Thanks for your contribution! Can you merge back `dev` to your branch to solve the conflicts? Thanks!

**Comment by jatisto** [comment]
Done!

File: src/main/plugins/browser-bookmarks-plugin/browser.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ export enum Browser {
     GoogleChrome = "Google Chrome",
     Brave = "Brave",
     Vivaldi = "Vivaldi",
+    SideKick = "SideKick"
```
Comment (oliverschwendener) [inline src/main/plugins/browser-bookmarks-plugin/browser.ts:5]: Missing comma here

File: src/main/production/production-search-engine.ts [hunk 2]
```diff
@@ -149,6 +154,7 @@ export function getProductionSearchEngine(
               new GoogleChromeBookmarkRepository(chromeBookmarksFilePath),
               new BraveBookmarkRepository(braveBookmarksFilePath),
               new VivaldiBookmarkRepository(vivaldiBookmarksFilePath),
+              new SideKickBookmarkRepository(sideKickBookmarkFilePath)
```
Comment (oliverschwendener) [inline src/main/production/production-search-engine.ts:157]: Missing comma here?

**Pull request merged**